config/database.yml

```diff
@@ -13,8 +13,8 @@ default: &default
   adapter: mysql2
   encoding: utf8mb4
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
```

Comment:
add this file to git ignore

Reply:
This file is already in .gitignore but somehow does not get ignored